logging/levels.go

```diff
@@ -77,9 +77,9 @@ func LookupLevel(name string) (slog.Level, error) {
 		return LevelInfo, nil
 	case levelNoticeName:
 		return LevelNotice, nil
-	case levelWarningName:
+	case levelWarningName, "WARN":
```

Comment:
Should the same pattern be applied to the other levels? Is there a reason for these two to behave differently?

Reply:
This is to maintain backwards compat with the old loggers levels. The other levels remain the same. I can document that if desired.

Reply:
Yeah, might be worth a note. I don't have strong feelings on it.

Reply:
Added docs.